pkgs/by-name/sy/sysdig-cli-scanner/package.nix

```
@@ -0,0 +1,55 @@
+{
+  stdenv,
+  lib,
+  fetchurl,
+  makeWrapper,
+}:
+let
+  versionMetadata = import ./sysdig-cli-scanner.versions.nix;
+  fetchForSystem = versionMetadata.${stdenv.system} or (throw "unsupported system ${stdenv.system}");
+in
+stdenv.mkDerivation {
+  pname = "sysdig-cli-scanner";
+  version = versionMetadata.version;
+
+  src = fetchurl { inherit (fetchForSystem) url hash; };
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  dontUnpack = true;
+
+  installPhase = ''
+    runHook preInstall
+
+    install -D $src $out/bin/sysdig-cli-scanner
```

I think being a bit more specific here could be helpful:
```suggestion
    install -Dm755 -T $src $out/bin/sysdig-cli-scanner
```

---

Sure, makes sense!